.github/workflows/pr-check.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: pr-check
+
+on: [pull_request]
```

Comment:
probably need to say PR against `gh-pages` branch ?

Reply:
Yes and we should also fix the file check as it will not work for new files being part of the PR

Reply:
why it won't work for files being in the PR ?

Reply:
the file should stay in the main branch as well and not in `gh-pages` branch probably ?

Reply:
understood. so yes it should replace the https links by files link before proceeding the check